File: Srims.Server.Business/Fund/FundDescendState.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Srims.Server.Business.Fund
{
    /// <summary>
    /// 经费下拨状态
    /// </summary>
    public enum FundDescendState
    {
        /// <summary>
        /// 未知状态
        /// </summary>
        Unknown = 0,
        /// <summary>
        /// 等待审核
        /// </summary>
        WaitingCensor = 2,
        /// <summary>
        /// 审核驳回
        /// </summary>
        Reject = 3,
        /// <summary>
        /// 审核通过
        /// </summary>
        Passed = 4,
        /// <summary>
        /// 分配完成
        /// </summary>
        AllocationCompleted = 5,
        /// <summary>
        /// 删除
        /// </summary>
        Delete = 6,
    }
}

File: Srims.WebSite/javascript/srims/fund/FundDescendState.js

if (!Srims.fund) 
    Ext.namespace('Srims.fund');

Ext.namespace('Srims.fund.fundDescendState');

Srims.fund.fundDescendState.WaitingCensor = 'WaitingCensor';
Srims.fund.fundDescendState.Reject = 'Reject';
Srims.fund.fundDescendState.Passed = 'Passed';
Srims.fund.fundDescendState.AllocationCompleted = 'AllocationCompleted';

Srims.fund.fundDescendStateRender = function(value){
    switch (value) {
        case 'WaitingCensor':
            return '等待审核';
        case 'Reject':
            return '审核驳回';
        case 'Passed':
            return '待分配';
        case 'AllocationCompleted':
            return '分配完成';
        default:
            return '未知';
    }
}
